src/client/app/+home/home.component.ts

```diff
@@ -1,12 +1,12 @@
 import { FORM_DIRECTIVES } from '@angular/common';
 import { Component } from '@angular/core';
 
-import { NameListService } from '../../shared/index';
+import { NameListService } from '../shared/index';
 
 @Component({
   selector: 'sd-home',
-  templateUrl: 'app/+home/components/home.component.html',
-  styleUrls: ['app/+home/components/home.component.css'],
+  templateUrl: 'app/+home/home.component.html',
```

Comment:
Just a random thought - we're using commonjs which means that we can use `moduleId` with `module.id`. Not sure yet, if we should do that or keep the path absolute. `module.id` doesn't work with systemjs where we should use `__moduleName` instead.

Do you have any thoughts?